ledger/sandbox/src/test/lib/scala/com/digitalasset/platform/store/dao/JdbcLedgerDaoCompletionsSpec.scala

```diff
@@ -21,6 +22,28 @@ private[dao] trait JdbcLedgerDaoCompletionsSpec extends OptionValues {
 
   behavior of "JdbcLedgerDao (completions)"
 
+  it should "return the expected completion for an accepted transaction" in {
+    for {
+      from <- ledgerDao.lookupLedgerEnd()
+      (offset, tx) <- storeCreateTransaction()
+      to <- ledgerDao.lookupLedgerEnd()
+      (_, response) <- ledgerDao.completions
+        .getCommandCompletions(from, to, tx.applicationId.get, Set(tx.submittingParty.get))
+        .runWith(Sink.head)
+    } yield {
+      val receivedOffset =
+        ApiOffset.assertFromString(response.checkpoint.value.offset.value.value.absolute.value)
```

Comment:
Please pull this out into a helper, with a meaningful name.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5072/commits/2f35ea00e91ac6dd081dc601dfd4ece98c9923b7